src/components/ImportJobs/ImportJobs.js

```diff
@@ -62,7 +62,7 @@ export class ImportJobs extends Component {
   };
 
   componentDidMount() {
-    this.fetchUploadDefinition();
+    setTimeout(() => { this.fetchUploadDefinition(); }, 1000);
```

Comment:
Cancel the timer in `componentWillUnmount()`, please.

Reply:
Done